config/sidekiq.yml

```diff
@@ -5,3 +5,6 @@
 :logfile: ./log/sidekiq.json.log
 development:
   :logfile: null
+:queues:
+  - default
+  - mailers
```

Comment:
might be premature to have a second queue?

Reply:
Ah, so this was just the ActionMailer default and I couldn't think of a good reason to override it. I'm happy to, if you think it would cause confusion :-).

Reply:
nah - seems fine if it's a default.